odoo/tools/misc.py

```diff
@@ -1608,13 +1608,16 @@ def format_decimalized_amount(amount: float, currency=None) -> str:
     return "%s %s" % (formated_amount, currency.symbol or '')
 
 
-def format_amount(env: Environment, amount: float, currency, lang_code: str | None = None) -> str:
+def format_amount(env: Environment, amount: float, currency, lang_code: str | None = None, trailing_zeroes: bool = True) -> str:
```

Comment:
Hello @odoo/rd-framework-py 

Can we have your opinion? I am not sure it is the best way to remove the digits. 🙂 

Thanks in advance! 🙂